raiden-ts/src/raiden.ts

```diff
@@ -840,6 +842,9 @@ export class Raiden {
     const pfs = !options.pfs
       ? undefined
       : decode(PFS, options.pfs, ErrorCodes.DTA_INVALID_PFS, this.log.info);
+    const expiration = !options.lockTimeout
+      ? undefined
+      : this.state.blockNumber + options.lockTimeout;
```

Comment:
You state in the header what the default of this value is. But you don't set it here, rather somewhere else. It took me a couple of minutes to search through the code base where this default is set. It is within the `makeAndSignTransfer$()` observable if I'm correct. Is there something that prevents us from defining this here and do not split it? :thinking:

Reply:
The reasoning for this is that the automatic value (when undefined) may be a little different at the time the `transfer.request` is processed (due to `concatMap`), and since it's automatic, I'd rather have it calculated at handling time, instead of at dispatch time like here (which, of course, isn't a concern if the `lockTimeout` is explicitly passed.
I'll add a comment to inform where the default is defined.